src/zcl_abapgit_objects.clas.abap

```diff
@@ -646,8 +646,11 @@ CLASS zcl_abapgit_objects IMPLEMENTATION.
 
     rt_results = zcl_abapgit_file_status=>status( io_repo ).
     DELETE rt_results WHERE match = abap_true.     " Full match
-    SORT rt_results BY obj_type ASCENDING obj_name ASCENDING.
-    DELETE ADJACENT DUPLICATES FROM rt_results COMPARING obj_type obj_name.
+    SORT rt_results
+      BY obj_type ASCENDING
+         obj_name ASCENDING
+         filename ASCENDING.
+    DELETE ADJACENT DUPLICATES FROM rt_results COMPARING obj_type obj_name filename.
```

Comment:
Not quite sure whether the change in files_to_deserialize is appropriate for all scenarios.

Reply:
I guess this is okay, but we need to take note